nitro-protocol/contracts/interfaces/IForceMove.sol

```diff
@@ -99,17 +93,17 @@ interface IForceMove {
      * @dev Finalizes a channel by providing a finalization proof.
      * @param largestTurnNum The largest turn number of the submitted states; will overwrite the stored value of `turnNumRecord`.
      * @param fixedPart Data describing properties of the state channel that do not change with state updates.
-     * @param appPartHash The keccak256 of the abi.encode of `(challengeDuration, appDefinition, appData)`. Applies to all states in the finalization proof.
-     * @param outcomeHash The keccak256 of the abi.encode of the `outcome`. Applies to all stats in the finalization proof.
+     * @param appData The keccak256 of the abi.encode of `(challengeDuration, appDefinition, appData)`. Applies to all states in the finalization proof.
```

Comment:
I think this comment needs updating -- `appData` is not the keccak256 of anything?

Reply:
You are definitely right. I changed `@param` name, but overlooked the description.